src/io/flutter/module/FlutterModuleBuilder.java

```diff
@@ -61,6 +65,15 @@ public void setupRootModel(ModifiableRootModel model) throws ConfigurationExcept
   }
 
   @Override
+  public ModuleWizardStep modifyStep(SettingsStep step) {
+    if (getModuleType() != null && step instanceof ProjectSettingsStep) {
```

Comment:
Not sure.  Looking at this now...

Maybe @alexander-doroshko will beat me to it though? 😉